sklearn/model_selection/_search.py

```diff
@@ -865,35 +918,63 @@ class GridSearchCV(BaseSearchCV):
             'params'             : [{'kernel': 'poly', 'degree': 2}, ...],
             }
 
-        NOTE that the key ``'params'`` is used to store a list of parameter
+        NOTE
+
+        The key ``'params'`` is used to store a list of parameter
         settings dict for all the parameter candidates.
 
         The ``mean_fit_time``, ``std_fit_time``, ``mean_score_time`` and
         ``std_score_time`` are all in seconds.
 
-    best_estimator_ : estimator
+        For multiple metric evaluation, the scores for all the scorers are
+        available in the ``cv_results_`` dict at the keys ending with that
+        scorer's name (``'_<scorer_name>'``) instead of ``'_score'`` as shown
+        above. ('split0_test_precision', 'mean_train_precision' etc.)
+
+    best_estimator_ : estimator or dict
         Estimator that was chosen by the search, i.e. estimator
         which gave highest score (or smallest loss if specified)
         on the left out data. Not available if refit=False.
 
-    best_score_ : float
+        For multimetric evaluation (when the ``scoring`` parameter is a dict/
+        list), this parameter is a dict mapping scorer names to the estimator
+        that gave the best score for that scorer.
+
+    best_score_ : float or dict
         Score of best_estimator on the left out data.
 
-    best_params_ : dict
+        For multimetric evaluation (when the ``scoring`` parameter is a dict/
```

Comment:
I think all you need to say is "only available in multi-metric setting if refit != False"